templates/monitoring/metric_tag_configuration.tf.tpl

```diff
@@ -0,0 +1,86 @@
+{{- $_ := stencil.ApplyTemplate "skipIfNotService" -}}
+## We use a locals block to have a single point of truth for the metric names, types, and tags that we want all metrics to have
+locals {
+  # Defines the metric names and types that we want to limit tags for
+  metrics = {
+    ## Go metrics
```

Comment:
How much are these metrics actually being used? Picking a particularly large one (`go_gc_heap_allocs_by_size_bytes_total`), it seems like it isn't being used _at all_ in dashboards or anything.

These metrics all-told account for about 10% of our Datadog custom metric spend - so eliminating them could save us about 50k per year.

These all look like low-level information that's only going to be interesting in edge-case debugging. Can we just flag these off, and less customers enable them when they _really_ need them for live debugging?